src/h3lib/lib/latLng.c

```diff
@@ -137,6 +137,24 @@ double constrainLng(double lng) {
     return lng;
 }
 
+/**
+ * Normalize an input longitude according to the specified normalization
+ * @param  lng           Input longitude
+ * @param  normalization Longitude normalization strategy
+ * @return               Normalized longitude
+ */
+double normalizeLng(const double lng,
+                    const LongitudeNormalization normalization) {
+    switch (normalization) {
+        case NORMALIZE_NONE:
+            return lng;
+        case NORMALIZE_EAST:
+            return lng < 0 ? lng + (double)M_2PI : lng;
+        case NORMALIZE_WEST:
+            return lng > 0 ? lng - (double)M_2PI : lng;
+    }
```

Comment:
[GCC is complaining that there's no `default` case here](https://github.com/uber/h3/actions/runs/6658832157/job/18096562714?pr=785#step:6:71) so it's possible to reach the end of the function without returning a number.

Perhaps this could be rewritten as:

```suggestion
    switch (normalization) {
        case NORMALIZE_EAST:
            return lng < 0 ? lng + (double)M_2PI : lng;
        case NORMALIZE_WEST:
            return lng > 0 ? lng - (double)M_2PI : lng;
        default:
            return lng;
    }
```

Reply:
Updated, thanks